app/views/api.js

```diff
@@ -38,21 +53,37 @@ module.exports = exports = state => parseApiFile(state.params.wildcard).then(res
           <ul class="api__toc__search__results"></ul>
         </div>
 
-        <ul class="api__toc__items" id="toc" role="navigation">${raw(createApiSidebar(response).join(""))}</ul>
+        <ul class="api__toc__items" id="toc" role="navigation">${createApiSidebar(response)}</ul>
       </aside>
-
-      <section class="api__content" id="toc-content">${raw(createApiContent(response).join(""))}</section>
+      <section class="api__content">
+        ${createApiHeader(state.params.wildcard)}
+        <div class="api__documentation" id="toc-content">
+          ${createApiContent(response)}
+        </div>
+      </section>
     </div>
 
     <script src="/assets/scripts/plugins/jets.js"></script>
-    ${raw(apiScripts)}
+    <script src="/assets/scripts/api.js"></script>
   `;
+}).catch(() => {
+  redirectOr404(state.href);
 });
 
 
 
+
 //  H E L P E R S
 
+function createApiHeader(slug) {
```

Comment:
In master, `lbry.tech/api` defaults to displaying one of these pages. In this branch, that page is scrunched. There should be a redirect for this use case if a default API page isn't going to be used.

Reply:
/api now goes to /resources